data/categories.yaml

```diff
@@ -12,46 +10,46 @@ categories:
 #  - name: Cloud
 #    id: cloud
 #    description: Developing and Running a WildFly application on the cloud
-  - name: Clustering
-    id: clustering
-    description: Clustering, including Infinispan and JGroups
-  - name: Datasources
-    id: datasource
-    description: Server-provided `javax.sql.Datasource`s
-  - name: Domain Mode
-    id: domain-mode
-    description: Centralized multi-server management
+- name: Clustering
+  id: clustering
+  description: Clustering, including Infinispan and JGroups
+- name: Datasources
+  id: datasource
+  description: Server-provided javax.sql.Datasources
+- name: Domain Mode
+  id: domain-mode
+  description: Centralized multi-server management
 #  - name: Embedded
 #    id: embedded
 #    description: Embedding WildFly in another process
 #  - name: IIOP
 #    id: iiop
 #    description: Internet InterORB Protocol and CORBA
-  - name: Jakarta Batch
-    id: batch
-    description: Jakarta Batch
-  - name: Jakarta CDI / Weld
-    id: weld
-    description: Jakarta Contexts and Dependency Injection
-  - name: Jakarta Concurrency
-    id: concurrency
-    description: Jakarta Concurrency
-  - name: Jakarta Connectors
-    id: jca
-    description: Jakarta Connectors
-  - name: Jakarta EE
-    id: ee
-    description: Jakarta Enterprise Edition
-  - name: Jakarta Enterprise Beans
-    id: ejb
-    description: Jakarta Enterprise Beans (EJB)
+- name: Jakarta Batch
+  id: batch
+  description: Jakarta Batch
+- name: Jakarta CDI / Weld
+  id: weld
+  description: Jakarta Contexts and Dependency Injection
+- name: Jakarta Concurrency
+  id: concurrency
+  description: Jakarta Concurrency
+- name: Jakarta Connectors
+  id: jca
+  description: Jakarta Connectors
+- name: Jakarta EE
+  id: ee
+  description: Jakarta Enterprise Edition
+- name: Jakarta Enterprise Beans
+  id: ejb
+  description: Jakarta Enterprise Beans (EJB)
 #  - name: Jakarta Faces
 #    id: jsf
 #    description: Jakarta Faces
-  - name: Jakarta Mail
-    id: mail
-    description: Mail
-  - name: Jakarta Persistence
+- name: Jakarta Mail
+  id: mail
+  description: Mail
+- name: Jakarta Persistence
```

Comment:
Please comment out this line. This is a flaw in main, not this PR, but we might as well fix it here.